css/style.css

```
@@ -1,324 +1,7 @@
-    }
-
-    .fig{
-        margin-left: auto;
-        margin-right: 10%;
-    }
-    
-    
-}
+@import "./styles/navbar.css";
+@import "./styles/header.css";
+@import "./styles/section.css";
 
-
-
-
-@media screen and (max-width: 800px){
-    .big-screen-nav{
-        display: none;
-    }
-    .small-screen-nav{
-        display: flex;
-        width: 100%;
-        margin-top: 20px;
-        margin-bottom: 20px;
-    }
-
-    .logo{
-        padding: 5px;
-        justify-content: flex-end;
-        margin-left: auto;
-        margin-right: 0;
-    }
-
-    .logo > img{
-        width: 80px;
-    }
-
-    .list{
-        display: inline-block;
-        width: 70%;
-        justify-content: flex-start;
-    }
-
-    .list > img{
-        width: 25px;
-    }
-
-    .items{
-        padding-left: 10px;
-        padding-right: 10px;
-        margin-left: 10px;
-        margin-right: 10px;
-        font-size: 90%;
-    }
-
-    .small-screen-header{
-        width: 80%;
-        display: flex;
-        flex-wrap: wrap-reverse;
-        margin-left: auto;
-        margin-right: auto;
-        padding: auto;
-        background-color: rgb(249 249 249);
-    }
-
-    .content{
-        align-items: center;
-        display: inline-block;
-        background-color: white;
-        width: 100%;
-        font-size: 150%;
-    }
-
-    .fig{
-        align-items: center;
-        display: flex;
-        width: 100%;
-        justify-content: center;
-    }
-
-    .fig > img{
-        display: inline-block;
-    }
-}
-
-
-.courses{
-    width: 70%;
-    margin-left: auto;
-    margin-right: auto;
-}
-
-.courses > h2{
-    font-size: 200%;
-}
-
-.courses > p{
-    font-size: 150%;
-    margin-bottom: 2%;
-}
-
-.explain{
-    border: 1px solid rgb(188, 186, 186);
-    padding: 2%;
-}
-
-.explain > h3{
-    font-size: 140%;
-    margin-bottom: -5px;
-}
-.explain > p{
-    width: 70%;
-    font-size: 110%;
-}
-
-.explain > a{
-    display: inline-block;
-    border: 2px solid black;
-    padding: 15px;
-    margin: 1%;
-    text-decoration: none;
-}
-
-.explain > a:hover{
-    color: black;
-    background-color: rgb(235, 235, 235);
-    text-decoration: none;
-}
-
-.explain > a:visited{
-    color: black;
-    background-color: rgb(235, 235, 235);
-    text-decoration: none;
-}
-
-
-.container{
-    display: flex;
-    flex-wrap: wrap;
-    justify-content: center;
-}
-
-.container > a{
-    display: flex;
-    width: 250px;
-    justify-content: center;
-    text-decoration: none;
-}
-
-.container > a:hover{
-    color: rgb(134, 131, 131);
-    background-color:rgb(226, 226, 226);
-    text-decoration: none;
-}
-
-.container > a:visited{
-    color: black;
-    text-decoration: none;
-}
-
-
-
-.course{
-    width: 100%;
-    align-items: center;
-    margin: 5px;
-}
-
-.thumb{
-    display: inline-block;
-    width: 100%;
-    margin-bottom: -15px;
-}
-
-.rate{
-    margin-top: -15px;
-    width: 100px;
+body{
+    justify-items: center;
 }
```

add a blank line at the end of the file.
```suggestion
}

```